fluent-package/yum/pkgsize-test.sh

```diff
@@ -67,7 +69,7 @@ done
 PREVIOUS_SIZE=$(stat -c %s $BASE_NAME)
 THRESHOLD_SIZE=`echo "$PREVIOUS_SIZE * 1.3" | bc -l | cut -d. -f1`
 find $REPOSITORIES_DIR/${DISTRIBUTION} -name td-agent-*.rpm
```

Comment:
Sorry, I guess I just don't get it, but why leave the `td-agent` here?

Reply:
It should be removed because it is just a debug print.

Reply:
Oh, I see. Thanks.